.github/workflows/snapshots.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020-2021 Dave Verwer, Sven A. Schmidt, and other contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Snapshots
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  snapshots:
+    name: Image Snapshots
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Build Front-End
+        run: make build-front-end
+      - name: Copy Front-End
+        run: cp -a ./Public/* ./Tests/AppTests/__Snapshots__/WebpageSnapshotTests
+      - name: Diff for HTML Changes
+        id: front-end-changes
+        run: |
+          filecount=$(git diff origin/main --shortstat "**.html" "**.css" "**.js" | awk '{print $1}')
+          echo "::set-output name=matching-files::${filecount:-0}"
+      - name: Run Percy
+        if: steps.front-end-changes.outputs.matching-files != '0'
+        run: npx @percy/cli snapshot ./Tests/AppTests/__Snapshots__/WebpageSnapshotTests
```

Comment:
minor: I'd probably keep the verbose flag in for now while we're still learning and fixing issues. Don't think it actively hurts us to have on.

Reply:
Good idea. 👍